package.json

```diff
@@ -22,59 +22,56 @@
     "check": "dependency-check . --no-dev --ignore-module patch-package --ignore-module websocket-reconnector",
     "coverage": "nyc --lines 95 --functions 95 --branches 95 --reporter=lcov --reporter=text npm test",
     "lint": "eslint --cache .",
-    "precommit": "npm run lint",
-    "prepare": "patch-package",
-    "prepublishOnly": "npm run precommit && npm run prepush && tag-matches",
-    "prepush": "npm run check && npm test",
+    "prepare": "patch-package && husky install",
+    "prepublishOnly": "npm run lint && npm run check && npm test && tag-matches",
     "test": "mocha"
   },
   "dependencies": {
-    "axios": "0.18.1",
-    "axios-cookiejar-support": "^0.4.2",
+    "axios": "0.21.1",
+    "axios-cookiejar-support": "^1.0.1",
     "debug": "^4.1.1",
-    "ethereumjs-wallet": "^0.6.3",
+    "ethereumjs-wallet": "1.0.1",
     "lodash": "^4.17.15",
     "merkletreejs": "0.0.25",
     "metronome-contracts": "^2.5.0",
-    "metronome-sdk-status": "^1.2.0",
-    "p-all": "^2.1.0",
-    "p-defer": "^1.0.0",
-    "p-retry": "^3.0.1",
-    "p-timeout": "^2.0.1",
+    "metronome-sdk-status": "^1.2.1",
+    "p-all": "^3.0.0",
+    "p-defer": "3.0.0",
```

Comment:
while I was able to replace most of `deferred` preferences with a native `Promise`, there's one scenario which has a weird comment [here](https://github.com/autonomoussoftware/metronome-wallet-core/blob/master/src/plugins/explorer/log-transaction.js#L7).

> // PromiEvent objects shall be wrapped to avoid the promise chain to
> // cast it to a plain promise

can (and should) I replace this one with a native Promise too?

Reply:
@gndelia I think the `deferred` could be replaced there too.

Reply:
using native promises https://github.com/autonomoussoftware/metronome-wallet-core/pull/34/commits/97ba5ec88025f0179f99bbedef6ebbbb7e1295e7